libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -138,7 +138,7 @@ function(create_libc_unittest fq_target_name)
 
   _get_common_test_compile_options(compile_options "${LIBC_UNITTEST_C_TEST}"
                                    "${LIBC_UNITTEST_FLAGS}")
-  list(APPEND compile_options ${LIBC_UNITTEST_COMPILE_OPTIONS})
+  list(APPEND compile_options ${LIBC_UNITTEST_COMPILE_OPTIONS} -static)
```

Comment:
If this is needed for your use-case, it might be better to just provide it in a CMake cache file. We should have an option to pass in user-requested flags right?

Reply:
Fixed.